### DISCUSSION TITLE
Accessing SSH recorded sessions

### DISCUSSION THREAD

**Discussion by asakth22** [gravitational/teleport#7201] [Q&A] [answered]
We've a Teleport node running through which we connect our client machines using TSH & SSH.
The recordings of these session shall be provided to our clients later if they ask.
So how can we achieve this ?

**Comment by webvictim** [comment, marked as answer]
To enable recording of traffic to SSH servers you'll need to use Teleport's proxy recording mode. See the docs here: https://goteleport.com/docs/server-access/guides/openssh/#set-up-openssh-recording-proxy-mode
